packages/bpk-mixins/src/mixins/_theming.scss

```diff
@@ -0,0 +1,28 @@
+$vars: (
+  primary-color: $bpk-button-background-color,
```

Comment:
Given how generic this now is we should directly refernce the lowelst level abstractions i.e color, but I can fix that late